Hmm... Build error is not related to the change:

```
npm ERR! network read ECONNRESET
npm ERR! network This is most likely not a problem with npm itself
npm ERR! network and is related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
```

--- a/package.json
+++ b/package.json
@@ -128,7 +128,7 @@
     "language-css": "0.32.2",
     "language-gfm": "0.80.0",
     "language-git": "0.10.0",
-    "language-go": "0.32.0",
+    "language-go": "0.34.0",
     "language-html": "0.40.1",
     "language-hyperlink": "0.14.0",
     "language-java": "0.16.0",

